src/main/java/seedu/address/model/widget/WidgetObject.java

```diff
@@ -40,140 +43,156 @@ public WidgetObject() {
+        return Optional.ofNullable(divs[2]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Div2.
      */
-    public Optional<String> divTwo() {
-        return Optional.ofNullable(divs[3]);
+    public String divTwo() {
+        return Optional.ofNullable(divs[3]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Text2.
      */
-    public Optional<String> textTwo() {
-        return Optional.ofNullable(divs[4]);
+    public String textTwo() {
+        return Optional.ofNullable(divs[4]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Div3.
      */
-    public Optional<String> divThree() {
-        return Optional.ofNullable(divs[5]);
+    public String divThree() {
+        return Optional.ofNullable(divs[5]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Text3.
      */
-    public Optional<String> textThree() {
-        return Optional.ofNullable(divs[6]);
+    public String textThree() {
+        return Optional.ofNullable(divs[6]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Div4.
      */
-    public Optional<String> divFour() {
-        return Optional.ofNullable(divs[7]);
+    public String divFour() {
+        return Optional.ofNullable(divs[7]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Text4.
      */
-    public Optional<String> textFour() {
-        return Optional.ofNullable(divs[8]);
+    public String textFour() {
+        return Optional.ofNullable(divs[8]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Div5.
      */
-    public Optional<String> divFive() {
-        return Optional.ofNullable(divs[9]);
+    public String divFive() {
+        return Optional.ofNullable(divs[9]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Text5.
      */
-    public Optional<String> textFive() {
-        return Optional.ofNullable(divs[10]);
+    public String textFive() {
+        return Optional.ofNullable(divs[10]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Div6.
      */
-    public Optional<String> divSix() {
-        return Optional.ofNullable(divs[11]);
+    public String divSix() {
+        return Optional.ofNullable(divs[11]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Text6.
      */
-    public Optional<String> textSix() {
-        return Optional.ofNullable(divs[12]);
+    public String textSix() {
+        return Optional.ofNullable(divs[12]).orElse("");
     }
 
     /**
      * Getter for 'attributes'.
      *
      * @return Footer.
      */
-    public Optional<String> footer() {
-        return Optional.ofNullable(divs[13]);
+    public String footer() {
+        return Optional.ofNullable(divs[13]).orElse("");
     }
```

Comment:
I feel like optional isn't needed in this file? It would be clearer if you do divs[X] == null ? "" : divs[X]

Reply:
Changed!